bins/packages/storcli/storcli.sh

```diff
@@ -1,17 +1,18 @@
+STORCLI_VERSION="1.21.06"
+
 dependencies_storcli() {
     apt-get update
-    apt-get install wget
 
     echo "[+] updating soruces list"
     wget -O - http://archive.thomas-krenn.com/tk-archive.gpg.pub | apt-key add -
-    cd /etc/apt/sources.list.d
-    wget http://archive.thomas-krenn.com/tk-main-xenial.list
-    wget http://archive.thomas-krenn.com/tk-optional-xenial.list
+
+    wget -O /etc/apt/sources.list.d/tk-main-xenial.list http://archive.thomas-krenn.com/tk-main-xenial.list
+    wget -O /etc/apt/sources.list.d/tk-optional-xenial.list http://archive.thomas-krenn.com/tk-optional-xenial.list
     cd
     apt-get update
 
     echo "[+] installing storcli"
-    apt-get install storcli
+    apt-get install storcli=${STORCLI_VERSION}
```

Comment:
This could break if package is not available on the ubuntu repository anymore.

You should download it from: https://docs.broadcom.com/docs-and-downloads/raid-controllers/raid-controllers-common-files/007.1316.0000.0000_Unified_StorCLI_PUL.zip

This is the last version, more up-to-date than Ubuntu version (`1.21.06 Nov 18, 2016` vs `007.1316.0000.0000 Mar 12, 2020`)

Extract the zip, extract the `Unified_storcli_all_os/Linux/storcli-007.1316.0000.0000-1.noarch.rpm` and take the binary from there.